.eslintrc

```diff
@@ -22,6 +22,6 @@
   },
 
   "rules": {
-    "comma-dangle": ["always-multiline"]
+    "comma-dangle": [2, "always-multiline"]
```

Comment:
Great, that worked! There are still a few lint errors due to unused or non define vars in different files previous to this PR
@wilsonpage what do you prefer me to do, fixing them in this PR or in another (to not create a big mixed PR)
Thanks!